### DISCUSSION TITLE
ASA Stats related topics in DMs

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#6] [General]
This is an example message from our Twitter DM:

-----

> Hello do we have an ETA for staking options on yieldly or Algostake? Is this still on the table?

We can't provide any information for Yieldly besides https://yieldly.medium.com/a-note-from-the-founder-2-a-hard-months-work-pays-off-8eb750331fb9
Our initial discussion for Algostake hadn't outcome with a decision to join their program.

-----

What do you think about how we should approach questions in Twitter DM like this one? Or in the case of my Discord or Reddit DM, or any other prominent community member's?

What would be the best way to expose them on Channel?

**Comment by Zamanikings** [comment]
I think this works.... But if subsequent suggestions are needed... We would surely say something about it

**Comment by wxivd** [comment]
**A: We value transparency and decentralisation
B: We need agility** 

If we are to be **pragmatic** about dealing with DMs, I suggest the following:

### Leave in DMs everything that has been addressed before
* Seeking information: we guide them to available resources: [Whitepaper](https://www.asastats.com/whitepaper.pdf), [Roadmap](https://github.com/asastats/docs/blob/main/roadmap.md), [ASA Stats Channel](https://github.com/tinymanorg/asa-list), [Discord server](https://discord.gg/Vjx7w7pAC7), [subreddit](https://www.reddit.com/r/asastats/), and [Twitter](https://twitter.com/asastatscom).
* Reporting bugs / requesting features: we assure it’s on out to-do-list [Channel issues](https://github.com/asastats/channel/issues).
* Business proposal / Collaboration request: we guide them to [ASA Stats Channel](https://github.com/asastats/channel).

### Leave in DMs everything that does not require discussion or further action
- Yes / No / Can not provide / Yet to be defined answers 

### Reveal on Channel
* New and novel ideas
* Not addressed issues or questions for discussions

> What would be the best way to expose them on Channel?
- Open [discussions](https://github.com/asastats/channel/discussions), Create [issues](https://github.com/asastats/channel/issues).
- Reference / give credit to the source (user or community member).
